Add improvements. (#2662)
wso2/docs-is: en/docs/references/concepts/claim-management.md

anything that the subject is owned by or associated with, such as name,
group, preferences, etc. A claim provides a single and general notion to
define the identity information related to the subject. Claim-based
identity is a common way for any application to acquire identity
information. It provides a consistent approach for all applications by
hiding the lower level implementation. Claims are also used in identity
propagation which is the replication of authenticated identities,
by packaging the claims into one or more tokens (such as SAML). These
tokens are then issued by an issuer (eg., [security token service](../authentication/intro-ws-trust)).

A user claim is a claim that is related to a user. It can be used to
specify information that is directly related to the user, such as claims
related to the street address, username, email, first name, and more.

An identity claim is a special claim related to identity management. It
can be used to specify information about the user account or the state

> of a user account such as the lastLoginTime, accountDisabled, and

Reviewer: ```suggestion
of a user account such as the `lastLoginTime`, `accountDisabled`, and
```

> accountLocked claims. Identity claims are identified by the claim URI.

Reviewer: ```suggestion
`accountLocked` claims. Identity claims are identified by the claim URI.
```

All identity claims have the term "identity" appended to the claim URI
as follows:
`http://wso2.org/claims/identity/accountDisabled`

> The Claim Management component of [WSO2

Reviewer: I don't think we maintain the carbon docs anymore. Better to not give this link.

Carbon](https://docs.wso2.com/display/Carbon4411/WSO2+Carbon+Documentation)
allows you to map a set of attributes from the underlying user store to
a set of defined claims. The underlying user store can either be an
internal or external JDBC user store, Active Directory, or LDAP user
store. You can [configure the primary user store](../../../deploy/configure-the-primary-user-store#set-up-the-primary-userstore) 

> using the deployment.toml file. 

Reviewer: ```suggestion
using the `deployment.toml` file. 
```

Each claim can be uniquely identified within the claim dialect by the

> Claim URI. Claim URIs are independent of the user store and each claim

Reviewer: ```suggestion
Claim URI. Claim URIs are independent of the userstore and each claim
```

> URI can be mapped into any desired attribute in the user store.

Reviewer: Please replace user store with userstore everywhere.

Therefore, at the application level we would know about the claims, but
not the attribute of the user store. An advantage of this is that we do
not need to be concerned about the user store level when we develop an
application as it is hidden by the claim management.

!!! info 
    In the case of every tenant startup, including the super tenant, the claim
    configurations are read directly from the

>     `<IS_HOME>/repository/conf/claim-config.xml` file. So

Reviewer: ```suggestion
    the `<IS_HOME>/repository/conf/claim-config.xml` file. So
```

    claims mapped for SCIM in the super tenant's [management
    console](../../../deploy/get-started/get-started-with-the-management-console) are not seen by
    a newly created tenant as the [User
    Realm](../../../deploy/configure-the-system-administrator) is always built using the

>     configuration found in the claim-config.xml file. So the recommended

Reviewer: ```suggestion
    configuration found in the `claim-config.xml` file. So the recommended
```

Please check if all file names are in monospace.
